daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -845,7 +845,7 @@ object Main extends Logging {
               case Some("-") | None => System.out
               case Some(file) => new FileOutputStream(file)
             }
-            val writer = new BufferedWriter(new OutputStreamWriter(output))
+            val writer = new BufferedWriter(new OutputStreamWriter(output, "UTF-8"))
```

Comment:
I tested this by changing my settings to use DAFFODIL_JAVA_OPTS="-Dfile.encoding=ASCII". The "É" in my output showed as a "?" without this change. I'm not sure if there is a test we can add to show this though.

Reply:
I don't think a test is needed.